### DISCUSSION TITLE
Weather App

### DISCUSSION THREAD

**Discussion by VeryUsual** [InfiniTimeOrg/InfiniTime#1733] [Ideas]
It would be cool to have a weather app.

**Comment by VeryUsual** [comment]
https://codeberg.org/Freeyourgadget/Gadgetbridge/wiki/Weather

**Comment by FintasticMan** [comment]
InfiniTime already has a weather service to get and parse weather information, but there's no app to display it yet (except for the test app for debugging). There is an open pull request for weather icons in the PineTimeStyle watch face: #1459.
